### ISSUE TITLE
Discoverable Content QA: "tell search engines which version of a page to crawl"

### ISSUE THREAD

**Issue by egsweeny** [GoogleChrome/web.dev#379]
- [x] Crawling is how search engines update their index of content on the web, and by providing search engines information about your preferred canonical URL you're ________ doing what?
- [x] "Lighthouse displays the following failed audit if your content is difficult for search engines to understand." My content is difficult to understand how? We were just talking about URL duplication. Not about content. Hard to follow as a user.
- [x] Should put link for security content (HTTPS) stuff when 'HTTPS URLS instead of HTTP URLs' are mentioned.
- [ ] Robots meta element?
- [ ] Don't point the canonical URL to a different domain (needs a link for more information).
- [ ] No guidance is provided on how to choose which of your URLs is canonical. What makes a good or bad canonical URL? What informs that choice?
- [x] SP: 'element in the ______? Of the page
- [x] **BROKEN**: No content in 'Example' for option 1 in table beneath 'Specify the canonical link'=
- [x] The 'Verify' section - it's kindof like okay, well, if the results are bad, then what do I do? It doesn't say anything. I just says to check the results. No action if it is good or bad (move onto next guide, or double check steps XXX).

**Comment by robdodson** [comment]
Adding @Meggin as well in case she has time to do a pass on these docs.
